includes/class-wc-install.php

```diff
@@ -806,6 +806,20 @@ private static function get_schema() {
   orders_net_total double DEFAULT 0 NOT NULL,
   PRIMARY KEY (start_time)
 ) $collate;
+CREATE TABLE {$wpdb->prefix}wc_order_product_lookup (
+  order_id BIGINT UNSIGNED NOT NULL,
+  product_id BIGINT UNSIGNED NOT NULL,
+  order_item_id BIGINT UNSIGNED NOT NULL,
```

Comment:
Should `order_item_id` be the primary key? In theory, each row in this table should map to a unique order item from the `wp_woocommerce_order_items` table. Just asking; I'm not an expert. :)